healthcare/datasets/setDatasetIamPolicy.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019, Google, LLC
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-warning-comments */
+
+'use strict';
+
+function main(
+  projectId = process.env.GCLOUD_PROJECT,
+  cloudRegion = 'us-central1',
+  datasetId,
+  member,
```

Comment:
Can we add default values or comments for `id`, `member`, and `role` as well please?